data/categories.yml

```diff
@@ -0,0 +1,216 @@
+- title: Economic Abuse Defined in State Laws
+  help_text: We cannot begin to address economic abuse (which occurs in 99% of intimate partner violence cases) without properly defining it in state laws.
+  ideal_criteria:
+    - text: "The state’s definition of intimate partner violence includes all or similar language to the following: Committing acts of economic abuse, including but not limited to: controlling, regulating, monitoring or depleting the other party’s movements, finances, economic resources, credit, access to services, basic necessities, ability to work and/or ability to pursue education or job training"
+      id: 1
```

Comment:
I have manually assigned every criterion an id here because the titles are way too long to reliably copy-paste them when referring to them in the state files.